src/EPR.RegulatorService.Facade.Core/Services/Producer/ProducerService.cs

```diff
@@ -28,7 +29,12 @@ public async Task<HttpResponseMessage> GetOrganisationsBySearchTerm(Guid userId,
 
         _logger.LogInformation("Attempting to fetch organisations by searchTerm '{searchTerm}'", searchTerm);
 
-        return await _httpClient.GetAsync(url);
+        var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
Please undo this change, the uriBuilder.Path returns encoded Url and if there is ? in the Url it will be converted to 3f something, ignore the SQ warning as peter is happy to leave it as is